src/components/Reports/FinancialAccountsReport/AccountTransactions/AccountTransactionTable/AccountTransactionTable.tsx

```diff
@@ -297,15 +305,13 @@ export const AccountTransactionTable: React.FC<TableProps> = ({
     <>
       <StyledDataGrid
         rows={transactions}
-        rowCount={transactions.length}
         columns={columns}
-        pageSize={pageSize}
-        onPageSizeChange={(pageSize) => setPageSize(pageSize)}
-        rowsPerPageOptions={[25, 50, 100]}
+        paginationModel={paginationModel}
```

Comment:
How come you don't need `paginationMode` for this DataGrid?

Reply:
By default, the DataGrid does client-side pagination. That mode requires that all of the rows be loaded up-front. For this table, we do have all of the rows available after the first API request comes back.